src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -0,0 +1,104 @@
+@GCS_Source
+Feature: GCS source - Verification of GCS to GCS Additional Tests successful
+
+  @GCS_MULTIPLE_FILES_TEST @GCS_SINK_TEST @EXISTING_GCS_CONNECTION
+  Scenario: To verify the pipeline is getting failed from GCS Source to GCS Sink On Multiple File using connection
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "GCS" from the plugins list as: "Sink"
+    Then Connect plugins: "GCS" and "GCS2" to establish connection
+    Then Navigate to the properties page of plugin: "GCS"
+    Then Select dropdown plugin property: "select-schema-actions-dropdown" with option value: "clear"
+    Then Click plugin property: "switch-useConnection"
+    Then Click on the Browse Connections button
+    Then Select connection: "gcsConnectionName"
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Enter GCS source property path "gcsMultipleFilesPath"
+    Then Select GCS property format "delimited"
+    Then Enter input plugin property: "delimiter" with value: "delimiterValue"
+    Then Validate "GCS" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "GCS2"
+    Then Click plugin property: "switch-useConnection"
+    Then Click on the Browse Connections button
+    Then Select connection: "gcsConnectionName"
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
```

Comment:
Ideally we should create "sinkRef" to be used at sink Side .

Reply:
Passing the "sinkRef" value in the GCS Sink plugin properties.